tempodb/encoding/vparquet/compactor.go

```diff
@@ -36,8 +36,7 @@ func (c *Compactor) Compact(ctx context.Context, l log.Logger, r backend.Reader,
 		minBlockStart   time.Time
 		maxBlockEnd     time.Time
 		bookmarks       = make([]*bookmark, 0, len(inputs))
-		pool            = newRowPool(100_000)
-		prefetchSize    = c.opts.IteratorBufferSize / (len(inputs) + 1) // 1 for each input plus multiblock iterator
+		pool            = newRowPool(int(c.opts.FlushSizeBytes))
```

Comment:
this defaults to ~31M and we don't override it anywhere as far as I can tell. it feels weird to chuck this config in here.

does the per tenant max_bytes_per_trace make more sense?

Reply:
I think either makes sense, as long as it's large enough to prevent allocations for the largest traces.   Now that we have `max_bytes_per_trace` it does seem more appropriate. In the case that there is no limit, it would be zero.  Will change it.